glean-core/tests/common/mod.rs

```diff
@@ -25,3 +30,23 @@ pub fn new_glean() -> (Glean, tempfile::TempDir) {
 
     (glean, dir)
 }
+
+/// Convert an iso8601::DateTime to a chrono::DateTime<FixedOffset>
+pub fn iso8601_to_chrono(
+    datetime: &iso8601::DateTime,
+) -> Option<chrono::DateTime<chrono::FixedOffset>> {
```

Comment:
It's for tests only. Can we just not wrap it in option and instead `panic!` when we get something we can't handle? Would save us some unwraps further down.

Reply:
That's a good idea.